frontend/.gitignore

```diff
@@ -18,6 +18,4 @@
 .env.test.local
 .env.production.local
 
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+*.log
```

Comment:
Should there be multiple .gitignore files? They apply recursively, so adding *.log here is unnecessary.

Reply:
I believe this file was generated automatically by yarn/node/etc so I will revert to what it was originally.